components/io-libs/adios2/SPECS/adios2.spec

```diff
@@ -93,6 +93,8 @@ cmake \
     -DCMAKE_INSTALL_PREFIX=%{install_path} \
     -DCMAKE_BUILD_TYPE=Release \
     -DBUILD_SHARED_LIBS=ON \
+    -DCMAKE_SKIP_INSTALL_RPATH:BOOL=ON \
+    -DCMAKE_SKIP_RPATH:BOOL=ON \
```

Comment:
I wonder would it better if those two lines are wrapped in `%if "%{compiler_family}" == "arm1"` ?!
It was failing only for `arm1` and you said that RPATH is quite common in HPC world.